packages/shared/src/components/Feed.tsx

```diff
@@ -166,17 +166,9 @@ export default function Feed<T>({
     return <></>;
   }
 
-  // const {
```

Comment:
Can you please revert this for now.
We need it in the code for when we urgently need to revert back, it's still unclear if this removal has a big impact.

Perhaps you can add a comment that explains this, as I can see why you removed it.